langchain/client/langchain.py

```diff
@@ -383,6 +386,92 @@ def list_examples(
         raise_for_status_with_text(response)
         yield from [Example(**dataset) for dataset in response.json()]
 
+    def update_example(
+        self,
+        example_id: str,
+        inputs: Optional[Dict[str, Any]] = None,
+        outputs: Optional[Dict[str, Any]] = None,
+        dataset_id: Optional[str] = None,
+    ) -> Dict[str, Any]:
+        """Update a specific example."""
+        example = ExampleUpdate(
+            inputs=inputs,
+            outputs=outputs,
+            dataset_id=dataset_id,
+        )
+        response = requests.patch(
+            f"{self.api_url}/examples/{example_id}",
+            headers=self._headers,
+            data=example.json(exclude_none=True),
+        )
+        raise_for_status_with_text(response)
+        return response.json()
+
+    def create_feedback(
+        self,
+        run_id: str,
+        metric_name: str,
+        metric_value: Union[float, str],
+        *,
+        source_info: Optional[Dict[str, Any]] = None,
+    ) -> Feedback:
+        """Create a feedback in the LangChain+ API.
+
+        Args:
+            run_id: The ID of the run to provide feedback on.
+            metric_name: The name of the metric, tag, or 'aspect' this
+                feedback is about.
+            metric_value: The score to rate this run on the metric, or
+                the value or label to assign for this metric.
+            source_info: Information about the source of this feedback.
+            extra: Extra information to include with the feedback.
+        """
+        feedback_source = APIFeedbackSource(metadata=source_info)
+        feedback = FeedbackCreate(
+            run_id=run_id,
+            metric_name=metric_name,
+            metric_value=metric_value,
+            feedback_source=feedback_source,
+        )
+        response = requests.post(
+            self.api_url + "/feedback",
+            headers=self._headers,
+            data=feedback.json(),
+        )
+        raise_for_status_with_text(response)
+        return Feedback(**feedback.dict())
+
+    @retry(stop=stop_after_attempt(3), wait=wait_fixed(0.5))
```

Comment:
Are we able to specify which errors are retried? i.e., 5xx and a few selected 4xx

Reply:
Ya - I think I'll update that in a subsequent PR though because our error raising isn't super specific